tests/test.js

```diff
@@ -4,7 +4,6 @@ let should = chai.should();
 const jsdom = require("jsdom");
 const {JSDOM} = jsdom;
 
-//const {Zvezdochki} = require("../dist/Zvezdochki.3283b0c3");
 const {Zvezdochki} = require("../src/Zvezdochki");
```

Comment:
```suggestion
const Zvezdochki = require("../src/Zvezdochki");
```
But before you need to update with the master.